pkg/provision/providers/qemu/launch.go

```diff
@@ -505,3 +505,27 @@ func dumpIpam(config LaunchConfig) error {
 
 	return nil
 }
+
+func checkBootDiskPartition(config *LaunchConfig) (diskBootable bool, err error) {
```

Comment:
I would prefer not to introduce yet another dependency into the mix, and instead make `go-blockdevice` work good enough on Darwin. It should be easy, as we are probing a file, not a real block device